docs/release-notes/0.6/release-0.6.md

```diff
@@ -0,0 +1,63 @@
+# ML.NET 0.6 Release Notes
+
+Today we are excited to release ML.NET 0.6, the biggest release of ML.NET ever (or at least since 0.5)! This release unveils the first iteration of new ML.NET APIs. These APIs enable various new tasks that weren't possible with the old APIs. Furthermore, we have added a transform to get predictions from [ONNX](http://onnx.ai/) models, expanded functionality of the TensorFlow scoring transform, aligned various ML.NET types with .NET types, and more!
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET
+Core
+2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md)
+for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+* New APIs for ML.NET
+    
+    * While the `LearningPipeline` APIs that were released with ML.NET 0.1 were easy to get started with, they had obvious limitations in functionality. Certain tasks that were possible with the internal version of ML.NET like inspecting model weights, creating a transform-only pipeline, and training from an initial predictor could not be done with `LearningPipeline`.
+    * The important concepts for understanding the new API are introduced [here](https://github.com/dotnet/machinelearning/blob/3cdd3c8b32705e91dcf46c429ee34196163af6da/docs/code/MlNetHighLevelConcepts.md). 
```

Comment:
>3cdd3c8b32705e91dcf46c429ee34196163af6da [](start = 128, length = 40)

now that it's merged, you should use [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md) etc.